core/dbt/utils.py

```diff
@@ -631,3 +631,12 @@ def __getitem__(self, name: str) -> Any:
 
     def __contains__(self, name) -> bool:
         return any((name in entry for entry in self._itersource()))
+
+
+def sql_escape(string):
```

Comment:
I think this is more of a weird bigquery thing than an "SQL" thing. Can this just go in the biguery impl.py?